thoth/glyph/cli.py

```diff
@@ -93,18 +102,66 @@ def generate(output: str) -> None:
     "--message",
     "-m",
     type=str,
-    default="-",
+    required=True,
     help="Commit message to be classified",
 )
 def classify(message: str) -> None:
     """Generate CHANGELOG entries from the current Git project."""
     _LOGGER.info("Hello, glyph!")
-    if message == "-":
-        click.echo("Please enter message")
-    else:
-        classifier = load_model("../../data/model_commits_v2_quant.bin") 
-        label = classifier.predict(message)
-        click.echo("Label : " + str(label[0][0])[8:])
+    _LOGGER.info("Model Path : " + MODEL_PATH)
+    classifier = load_model(MODEL_PATH) 
+    label = classifier.predict(message.lower())
+    click.echo("Label : " + str(label[0][0])[8:])
+
+@cli.command()
+@click.option(
+    "--path",
+    "-p",
+    type=str,
+    required=True,
+    help="Path to Git repository"
+)
+@click.option(
+    "--start",
+    type=str,
+    default="",
+    help="Starting date"
+)
+@click.option(
+    "--end",
+    type=str,
+    default="",
+    help="End date"
+)
+def classifyrepo(path: str, start: str, end: str) -> None:
+    _LOGGER.info("Model Path : " + MODEL_PATH)
```

Comment:
```
    _LOGGER.info("Model Path: %r", MODEL_PATH)
```

It would be good to provide a path to the model using CLI arguments and default the one hardcoded (shipped with the package?) if not supplied.

Reply:
Fixed. Model added as an optional argument